ConfigurationSystem/Client/Helpers/CSGlobals.py

```diff
@@ -26,26 +75,13 @@ def getCSExtensions():
     Return list of extensions registered in the CS
     They do not include DIRAC
   """
-  from DIRAC import gConfig
-  return gConfig.getValue( '/DIRAC/Extensions', [] )
+  return Extensions().getCSExtensions()
```

Comment:
2 questions:
- Is this new helper class specific for v6r14? 
- The fact that it's been developed, does it mean that should be used everywhere in the code where gConfig.getValue( '/DIRAC/Extensions', [] ) is used? (I have this case in couple place in LHCb)